packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -540,6 +550,10 @@ export function TransferPanel() {
         )
       }
 
+      if (isSmartContractWallet) {
```

Comment:
This will not send analytics for SCW right?

Reply:
Good point, I have added the event for SCW. We assume for now that the transaction is going through